cmd/kube-proxy/app/server.go

```diff
@@ -557,7 +557,9 @@ func (s *ProxyServer) Run() error {
 	if s.CleanupAndExit {
 		encounteredError := userspace.CleanupLeftovers(s.IptInterface)
 		encounteredError = iptables.CleanupLeftovers(s.IptInterface) || encounteredError
-		encounteredError = ipvs.CleanupLeftovers(s.IpvsInterface, s.IptInterface, s.IpsetInterface, s.CleanupIPVS) || encounteredError
+		if s.CleanupIPVS {
```

Comment:
I don't think we need to check this here - the --cleanup flag is unilateral

Reply:
good point, fixed this